tools/star.wdl

```diff
@@ -87,17 +125,382 @@ task alignment {
+        clipAdapterType: {
+            description: "adapter clipping type",
+            choices: {
+                Hamming: "adapter clipping based on Hamming distance, with the number of mismatches controlled by --clip5pAdapterMMp",
+                CellRanger4: "5p and 3p adapter clipping similar to CellRanger4. Utilizes Opal package by Martin Šošić: https://github.com/Martinsos/opal",
+                None: "no adapter clipping, all other clip* parameters are disregarded"
+            }
+        }
+        outSAMstrandField: {
+            description: "Cufflinks-like strand field flag",
+            choices: {
+                None: "not used",
+                intronMotif: "strand derived from the intron motif. This option changes the output alignments: reads with inconsistent and/or non-canonical introns are filtered out."
+            }
+        }
+        outSAMattributes: {
+            description: "a string of desired SAM attributes, in the order desired for the output SAM. Tags can be listed in any combination/order. **[STAR defaults]**: `NH HI AS nM`. **[WDL default]**: `NH HI AS nM NM MD XS`.",
+            choices: {
+                NH: "number of loci the reads maps to: =1 for unique mappers, >1 for multimappers. Standard SAM tag.",
+                HI: "ultiple alignment index, starts with --outSAMattrIHstart (=1 by default). Standard SAM tag.",
+                AS: "local alignment score, +1/-1 for matches/mismateches, score* penalties for indels and gaps. For PE reads, total score for two mates. Standard SAM tag.",
+                nM: "number of mismatches. For PE reads, sum over two mates.",
+                NM: "edit distance to the reference (number of mismatched + inserted + deleted bases) for each mate. Standard SAM tag.",
+                MD: "string encoding mismatched and deleted reference bases (see standard SAM specifications). Standard SAM tag.",
+                jM: "intron motifs for all junctions (i.e. N in CIGAR): 0: non-canonical; 1: GT/AG, 2: CT/AC, 3: GC/AG, 4: CT/GC, 5: AT/AC, 6: GT/AT. If splice junctions database is used, and a junction is annotated, 20 is added to its motif value.",
+                jI: "start and end of introns for all junctions (1-based).",
+                XS: "alignment strand according to --outSAMstrandField.",
+                MC: "mate's CIGAR string. Standard SAM tag.",
+                ch: "marks all segment of all chimeric alingments for --chimOutType WithinBAM output.",
+                cN: "number of bases clipped from the read ends: 5' and 3'"
+            }
+        }
+        outSAMunmapped: {
+            description: "output of unmapped reads in the SAM format.",
+            choices: {
+                None: "no output **[STAR default]**",
+                Within: "output unmapped reads within the main SAM file (i.e. Aligned.out.sam) **[WDL default]**"
+            }
+        }
```

Comment:
I don't know that we strictly need to document the difference in the defaults here. I think `None` is nearly always the wrong option anyway.